src/coreComponents/mesh/generators/VTKMeshGenerator.cpp

```diff
@@ -230,35 +201,44 @@ vtkSmartPointer< vtkUnstructuredGrid > redistributeMesh( vtkUnstructuredGrid & l
  * @param[in] mesh the vtkUnstructuredGrid that is loaded
  * @return the global length of the mesh (diagonal of the bounding box)
  */
-double writeMeshNodes( CellBlockManager & cellBlockManager,
-                       vtkSmartPointer< vtkUnstructuredGrid > mesh )
+real64 writeMeshNodes( vtkUnstructuredGrid & mesh,
+                       R1Tensor const & scale,
```

Comment:
The scaling and translation of a mesh should be separated of the datastructure  conversion - the purpose of the VTKMeshGenerator.

If available in geosx (I would say it is the job of another software) thee scaling and translation should be applicable to all types of meshes -

Reply:
I agree - but if this is a VTK mesh couldn't this be done in Paraview? 

This is an operation exclusively on the coordinates of the vertices, right ? 
Maybe the NodeManager could do it ?